test/test_helper.rb

```diff
@@ -13,5 +13,25 @@ class TestCase
     fixtures :all
 
     # Add more helper methods to be used by all tests here...
+    def setup_omniauth_mock
+      OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new({
+                                                                           provider: 'google_oauth2',
+                                                                           uid: '12345678910',
+                                                                           email: 'mock@example.com',
+                                                                           info: {
+                                                                             name: 'mockuser',
+                                                                             image: 'https://test.com/test.png'
+                                                                           }
+                                                                         })
+    end
+
+    def sign_in
+      OmniAuth.config.mock_auth[:google_oauth2] = nil
+      Rails.application.env_config['omniauth.auth'] = setup_omniauth_mock
+    end
+
+    def signed_in
+      nil if Rails.application.env_config['omniauth.auth']
+    end
```

Comment:
`nil | void` が返ってる気がします。(env_configがtruthyであればnilを返し、そうでなければ何も返さない)
なので、 `if signed_in` は常に `false` になっている気がしましたがどうでしょうか。

おそらくやりたいのってこんな感じなのかなと感じました。

```suggestion
    def signed_in?
      !Rails.application.env_config['omniauth.auth'].nil?
    end
```

Reply:
おかしかった sign_in により、signed_in が機能するようになりました。
こいつも直しました。